### PR TITLE
Specified minimum cocoapods version in podspec to 1.10.0.

### PR DESCRIPTION
**Pull request by aleksandr-dorofeev**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* ~[ ] Are tests passing locally?~
* [x] Are the files formatted correctly?
*~[ ] Did you add unit tests?~
*~[ ] Did you check UI tests on the sample app? They are not executed on CI.~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Specified minimum cocoapods version in Appcenter.podspec file.

## Related PRs or issues

[AB88083](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/88083)

### CODE REVIEW

**Review by DmitriyKirakosyan — DISMISSED** [review]

File: AppCenter.podspec [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Pod::Spec.new do |s|
+  s.cocoapods_version = '>= 1.10'
```
Comment (AnastasiaKubova) [inline AppCenter.podspec:2]: do we need to update the changelog that we raised the minimal version of cocoapods?
Also please update the template in PR description.

**Review by AnastasiaKubova — CHANGES_REQUESTED** [review]

File: AppCenter.podspec (hunk 1 above)
Comment (aleksandr-dorofeev) [reply AppCenter.podspec:2]: done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ ___
 
 ## Version 4.3.0
 
+### App Center
+
+* **[Improvement]** Specified minimum cocoapods version in Appcenter 1.10 -> in podspec to 1.10.0
```
Comment (DmitriyKirakosyan) [inline CHANGELOG.md:11]:
```suggestion
* **[Improvement]** Specified minimum cocoapods version in podspec to 1.10.0.
```

**Review by DmitriyKirakosyan — APPROVED** [review]

File: AppCenter.podspec (hunk 1 above)
Comment (AnastasiaKubova) [reply AppCenter.podspec:2]: did you test your changes? why it doesn't mention in the template?

Comment (aleksandr-dorofeev) [reply AppCenter.podspec:2]: yes, now it's correct!

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Pull request merged**